LandSandBoat/server
# [Quest] Implement Elderly Pursuits

**_I affirm:_**

## What does this pull request do?

Adds the quest Elderly Pursuits, NM fight, and reward.

## Steps to test these changes
Have Secrets of Ovens Lost Quest Completed
Start the quest in Tav Stronghold by talking to Despachiaire !gotoid 16883721
-> Southern San d'Oria
Talk to Rouva !gotoid 17719492
-> Carpenter's Landing
Check QM to spawn Para NM
Kill the ORIGINAL PARA
Para will spawn clones when using specific tp moves when under 50% hp.
Check QM again after original Para is dead
-> Southern San d'Oria
Talk to Rouva !gotoid 17719492
-> Tavnavian Safehold
Talk to Despachiaire !gotoid 16883721 to end quest

##Sources
Retail Captures:
https://youtu.be/beJ9ca8aZYM
https://1drv.ms/u/s!AlLEVQXhZeaHgWSuTwo2iHks5-P1?e=fwwJVs
